src/libcmd/repl.cc

```diff
@@ -791,6 +807,27 @@ void NixRepl::addAttrsToScope(Value & attrs)
     staticEnv->sort();
     staticEnv->deduplicate();
     notice("Added %1% variables.", attrs.attrs()->size());
+
+    lastLoaded = attrs;
+
+    const int max_print = 10;
+    int counter = 0;
+    std::string loaded;
+    for (auto & i : attrs.attrs()->lexicographicOrder(state->symbols)) {
+        if (counter >= max_print)
+            break;
+
+        if (counter > 0)
+            loaded += ", ";
+
+        loaded += state->symbols[i->name];
```

Comment:
`printIdentifier` would be nice to use here.

Reply:
Indeed, that was the one I was looking for 😀